tests/cli/test_benchmark_audio_transcribe.py

```
@@ -0,0 +1,419 @@
+        assert getattr(result, "func", None) is _run_audio_suite
+        assert result.keywords == {"transcribe_smoke": True}
+
+
+@pytest.mark.unit
+@pytest.mark.ci
+class TestValidateTranscribeSmokePreconditions:
+    """Direct coverage of the empty-input / no-audio-candidates fail-fast
+    helper. Both paths used to short-circuit the benchmark before the
+    smoke-failure exit guard ran, exiting 0 with no verification done."""
+
+    def test_passes_silently_when_smoke_off(self, tmp_path: Path) -> None:
```

Ruff (0.15.12)

[warning] Unused method argument: `tmp_path`

(ARG002)